lib/pelemay.ex

```diff
@@ -39,67 +39,75 @@ defmodule Pelemay do
   defmacro defpelemay(functions) do
     Db.init()
 
+    caller_name = __CALLER__.module |> Generator.elixir_nif_module() |> String.to_atom()
+
     functions
-    |> SumMag.map(
-      &Optimizer.replace_expr(
-        &1,
-        __CALLER__.module |> Generator.elixir_nif_module() |> String.to_atom()
-      )
-    )
+    |> SumMag.map(&Optimizer.replace_expr(&1))
+    |> consist_context(caller_name)
     |> pelemaystub(__CALLER__.module)
   end
 
   defp pelemaystub(ret, module) do
     Generator.generate(module)
     ret
   end
+
+  defp consist_context(funcs, module) do
+    Macro.prewalk(
+      funcs,
+      fn
+        {:__aliases__, [alias: false], [:Temp]} -> module
```

Comment:
I think `Temp` is not suitable name.
We should adopt an alternative name after `Module`.

Reply:
How is not suitable? 
- Hard to read
- A cause of conflict